R/utils-shared.R

```diff
@@ -261,7 +261,8 @@ splitDataGaps <- function(data, ts, isDV){
       endGaps <- c(endGaps, endEstimated)
     }
     
-    if(isDV){ ts$time <- flexibleTimeParse(ts$time, timezone = data$reportMetadata$timezone) }
+    #This is causing DV values to be rendered at noon instead of on the day marks. Re-enable after AQCU 816?
```

Comment:
I try not to reference JIRA tickets in the source. The source code is public and the JIRA is not, so it provides little historical value.